tests/IdValidatorTest.php

```diff
@@ -12,28 +12,37 @@
 
 class IdValidatorTest extends TestCase
 {
+    /**
+     * @var \Jxlwqq\IdValidator\IdValidator
+     */
+    private $idValidator;
+
+    protected function setUp()
+    {
+        parent::setUp();
```

Comment:
Should not call the parent setUp because is  just a method template

Reply:
It's a common practice to always include it, as you might want to extend `\PHPUnit\Framework\TestCase`

Reply:
Hi @lucasmichot  ,
but the parent method seems to be empty method and will always be empty in phpunit (IMO).
If the parent method does nothing you don't need it  (YAGNI )

Reply:
As stated this is a common practice to include it, so that when you decide to create your own abstract TestCase extension (which always happen when the test base grows), you don't have to check all setUp and tearDown method.

YAGNI does not apply to tests when it enforces readability and consistency.

Reply:
If you feel the need to make a PR to remove it in case of a negative impact on the Unit tests suite, you're welcome to do so